### ISSUE TITLE
Any possibility to add support for Priority: Survive?

### ISSUE THREAD

**Issue by SenhorFlibble** [Wargus/stargus#47]
[Priority: Survive](https://www.allegro.cc/depot/PrioritySurvive/) is a free, but not open source, sci-fi RTS game that is in many ways similar to _StarCraft_. There are only Windows builds available (and only [via the Wayback Machine](https://web.archive.org/web/20180713035629/http://distortum.mysteria.cz/) or mirror sites because the official site of the game has apparently been taken down), and they do not really work properly on modern OSs: I was unable to run it correctly either on Windows 8.1 or under Linux through Wine.

I tried to contact the developers back when the site was still up (sometime in 2017-2018), and wrote an email asking if they would consider releasing the source code, but received no reply. Quite possibly, the developers have altogether abandoned this project.

Because _Priority: Survive_ is similar to _StarCraft_ both visually and gameplay-wise, I thought that perhaps it would be possible to incorporate it into the Stargus project? I do not know what game data format is uses, but apparently some StarCraft mods use sprites from PS, hopefully suggesting that data extraction is possible.

**Comment by timfel** [comment]
It's definitely possible, but a lot of work. As with anything open source - unless someone comes along who wants to do that work, it will not happen :-)

**Comment by yaaaaa** [comment]
"I was unable to run it correctly either on Windows 8.1 or under Linux through Wine."
+++
Anybody was can run it?

**Comment by SenhorFlibble** [comment]
> "I was unable to run it correctly either on Windows 8.1 or under Linux through Wine."
> +++
> Anybody was can run it?
Sure, you can find a number of videos on YouTube, like [this](https://www.youtube.com/watch?v=C3PcvcQ-w9k). The programme **does** run on both Windows 8.1 and Wine, it just does not work correctly. IIRC Windows has a garbled palette, while in Wine I had issues with the mouse pointer. I was able to make some screenshots for the [MobyGames entry](https://www.mobygames.com/game/priority-survive) but it was not really playable that way.

It might be that Wine compatibility has improved since then, as I played around with the game back in 2017, and cannot check in the current Wine version now. I also thought that maybe DxWnd or some such tool could fix the palette issues on Windows but again, haven't gotten around to actually try that out.

Generally it would be super nice to have a libre _StarCraft_ clone along the lines of Aleona's Tales, although I guess it'd take a lot more work to produce the necessary art. Since _Priority: Survive_ is free and seemingly abandoned by the developers, it could work as an interim free data set.

**Comment by timfel** [comment]
> Since Priority: Survive is free and seemingly abandoned by the developers, it could work as an interim free data set.

Unfortunately it being freeware does not imply that it's license allows using the assets for something else. Unless it explicitly permits that, we would probably still have to ask the authors. Since they removed the downloads from their official website, it's not clear what the state is there.

**Comment by SenhorFlibble** [comment]
> Unfortunately it being freeware does not imply that it's license allows using the assets for something else. Unless it explicitly permits that, we would probably still have to ask the authors. Since they removed the downloads from their official website, it's not clear what the state is there.

I understand that, but I don't think they'd object if Stargus had a data extractor/converted for PS assets similar to the one used to get proprietary Blizzard data. I did not suggest to just grab the free game data as the project's own.

As for the downloads having been removed, it seems to me that their domain ownership just expired.

Their [main site](http://distortum.net/) is still up and they're offering game art outsourcing services, apparently. But I checked my mailbox and I definitely sent my email to the address on their contact page at the distortum.net domain, without getting any reply.
